avm/res/service-fabric/cluster/main.bicep

```diff
@@ -447,6 +445,25 @@ output location string = serviceFabricCluster.location
 //   Definitions   //
 // =============== //
 
+type certificateType = {
```

Comment:
Thanks for adding this UDT! Could you please also define `certificateCommonNameType` for the same reason?

Reply:
Added UDT's for certificateCommonNamesType, serverCertificateCommonNameType, clientCertificateCommonNameType, and clientCertificateThumbprintType